### ISSUE TITLE
Transition not working when navigate back with finger gesture (IOS)

### ISSUE THREAD

**Issue by MarcHbb** [IjzerenHein/react-navigation-shared-element#260]
First, let me thank you for the work, this module is incredible.
I managed to get this work, from getting to a detail screen and go back (onPress) with the transition.  
But when I go back from using finger gesture (gesture from left of the screen to the right), the go back transition is not working.

**Comment by p-syche** [comment]
hi @MarcHbb ,

Thank you for the kind words ❤️ 
Can you provide a minimal repro or an expo snack of the issue you are seeing?

**Comment by shawarmaz** [comment]
yeah so I was seeing this too and to reproduce this, I really just pasted [the Usage snippet](https://github.com/IjzerenHein/react-navigation-shared-element/blob/main/docs/API.md#usage) into a snack

https://snack.expo.dev/3dgVG2POp

**Comment by shawarmaz** [comment]
example
![Screenshot 2023-02-01 at 4 42 31 AM](https://user-images.githubusercontent.com/81338316/215939991-0af3e461-2bb2-4b9d-9b6f-4870c905fd48.png)

**Comment by MarcHbb** [comment]
Hello @p-syche ! :) 

Sorry didn't get notified about answer on this thread ! Do you have some news or is it still in progress ?

**Comment by p-syche** [comment]
Hi @shawarmaz and @MarcHbb :)

Thank you for the repro @shawarmaz. I'm looking into this issue this week, I hope to get back to you with some info in a few days.

**Comment by bpeck81** [comment]
I'm seeing this issue

**Comment by paul12p-pichincha** [comment]
any update?
